src/liger_kernel/transformers/model/deepseekv2.py

```diff
@@ -0,0 +1,174 @@
+from typing import List
+from typing import Optional
+from typing import Tuple
+from typing import Union
+
+import torch
+
+from torch.nn import CrossEntropyLoss
+from transformers.modeling_outputs import CausalLMOutputWithPast
+from transformers.utils import add_start_docstrings_to_model_forward
+from transformers.utils import replace_return_docstrings
+
+from liger_kernel.transformers.fused_linear_cross_entropy import LigerFusedLinearCrossEntropyLoss
+
+DeepseekV2_INPUTS_DOCSTRING = r"""
```

Comment:
nit: it'll be helpful to document where this part of docstring is ported from -- at least need a link to https://huggingface.co/deepseek-ai/DeepSeek-V2/blob/main/modeling_deepseek.py